libs/langchain/langchain/chat_models/vertexai.py

```diff
@@ -19,7 +29,11 @@
     AsyncCallbackManagerForLLMRun,
     CallbackManagerForLLMRun,
 )
-from langchain.chat_models.base import BaseChatModel, _generate_from_stream
+from langchain.chat_models.base import (
+    BaseChatModel,
+    _agenerate_from_stream,
```

Comment:
@eyurtsev @baskaryan FYI Probably `_agenerate_from_stream` and `_generate_from_stream` could be promoted from the local functions to the library functions, no?